drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -290,6 +301,18 @@ def normalize_single_custom_field_value(cded, value)
       else
         raise "unexpected value \"#{value}\""
       end
+    when 'file'
+      case value
+      when String
+        value
```

Comment:
do we have any expectations about this string? it would be helpful to add a note explaining what we expect it to be.